Behaviour of add button in the editor is not OK for xml elements with a minimum and maximum cardinality equal to 1 #3010

gvandeboel opened the pull request:
See issue #3010

PascalLike commented:
Thank you for your contribution, @gvandeboel 
Can you please direct the pull request to the master branch?
The pull requests must be open to the **master** branch. If it's a bugfix for 3.4.x you can ask to backport it.
https://github.com/geonetwork/core-geonetwork/wiki/How-to-contribute

gvandeboel commented:
PascalLike, thank you for your advise.  I have created a new pull request on the master now and followed your advise.  Hopefully it will be accepted soon.  As you see, I have removed the linked branch on our github fork and cleaned up too all the internal issue links in the message of the commit.
